documentation/src/main/asciidoc/userguide/chapters/domain/basic_types.adoc

```diff
@@ -118,50 +80,33 @@ include::{sourcedir}/basic/ExplicitBasicTypeTest.java[tags=basic-annotation-expl
 ====
 
 [[basic-annotation-implicit-example]]
-.`@Basic` being implicitly implied
+.`@Basic` implied
 ====
 [source, JAVA, indent=0]
 ----
 include::{sourcedir}/basic/ImplicitBasicTypeTest.java[tags=basic-annotation-implicit-example]
 ----
 ====
 
-[TIP]
-====
-The JPA specification strictly limits the Java types that can be marked as basic to the following listing:
+The `@Basic` annotation defines 2 attributes.
 
-* Java primitive types (`boolean`, `int`, etc)
-* wrappers for the primitive types (`java.lang.Boolean`, `java.lang.Integer`, etc)
-* `java.lang.String`
-* `java.math.BigInteger`
-* `java.math.BigDecimal`
-* `java.util.Date`
-* `java.util.Calendar`
-* `java.sql.Date`
-* `java.sql.Time`
-* `java.sql.Timestamp`
-* `byte[]` or `Byte[]`
-* `char[]` or `Character[]`
-* `enums`
-* any other type that implements `Serializable` (JPA's "support" for `Serializable` types is to directly serialize their state to the database).
+`optional` - boolean (defaults to true):: Defines whether this attribute allows nulls.
+   JPA defines this as "a hint", which essentially means that its effect is specifically required.
```

Comment:
may be just my bad English but being an hint should not imply that the persistence provider can not take it into account? so its effect is **not** specifically required?

Reply:
Hm, I did actually not even notice this.  That was not "changed" from Vlad's original wording.  

I agree that could be made way more clear

Reply:
So I actually changed up both items to make it (imo) better English:

```


`optional` - boolean (defaults to true):: Defines whether this attribute allows nulls.  JPA defines
    this as "a hint", which means that the provider is free to ignore it (Hibernate does not ignore it).
    JPA also says that it will be ignored if the type is primitive.  Works in conjunction with
    `@Column#nullable` - see <<basic-column-annotation>>.
`fetch` - FetchType (defaults to EAGER):: Defines whether this attribute should be fetched eagerly or lazily.
    `EAGER` indicates that the value will be fetched as part of loading the owner.  `LAZY` values are
    fetched only when the value is accessed.  JPA requires providers to support `EAGER`, while support for
    `LAZY` is optional meaning that a provider is free to not support it.  Hibernate supports lazy loading
    of basic values as long as you are using its <<chapters/pc/BytecodeEnhancement.adoc#BytecodeEnhancement,bytecode enhancement>>
    support.
```

Comment on the next push if y'all don't like that wording.